src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetCurrentProcessId.cs

```diff
@@ -3,12 +3,14 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.InteropServices;
+using Internal.Runtime.InteropServices;
```

Comment:
Moving the attribute out of the shared partition but keeping the references in shared files will cause build breaks in Mono and CoreRT when this mirrors over. Why not just keep the attribute in the shared partition?

Reply:
Seconded, I actually plan to implement it for Mono.

Reply:
`Interop.GetCurrentProcessId.cs` is used in other CoreFX assemblies. If the attribute is internal and not exposed in public contracts, there are going to be build breaks in CoreFX and no straightforward way to fix them. If you would like to get this in before the API review discussion, I think it would be best to revert the example uses in CoreLib.

Reply:
The attribute name and documentation are more than likely going to change and I was attempting to keep it internal until the API review is complete. There is also the inevitable fallout of the name change that will occur involving updating various JIT and EE code locations that use the existing name. Keeping this internal makes it less of an issue for making these changes, at least that was my thought. If people aren't concerned about the eventual API name change I am more than willing to keep it public. Thoughts?

Reply:
Keeping it public in CoreLib is fine, but I do not think it should be put on Interop.* definitions in CoreLib until it is official. You would have to fight with CoreFX build breaks that is not worth it.